scripts/validate-packages.py

```diff
@@ -30,6 +33,11 @@ def _get_json_schema(file_name):
 def main():
     # traverse prefix dirs ("A", "B", etc)
     for letter in os.listdir(PKG_DIR):
+        if not LETTER_PATTERN.match(letter):
+            sys.exit(
+                "\tERROR\n\n"
+                "Invalid name for directory : {}".format(letter)
```

Comment:
Reword the error message to include the regex that the directory must match; in general, error messages should include information to help fix the problem, where possible.